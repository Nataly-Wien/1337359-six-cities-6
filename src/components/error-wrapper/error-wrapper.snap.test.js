import React from 'react';
import {render} from '@testing-library/react';
import ErrorWrapper from './error-wrapper';

it(`Should errors render correctly`, () => {
  const isLoadingError = true;

  const {container} = render(<ErrorWrapper isLoadingError={isLoadingError} />);

  expect(container).toMatchSnapshot();
});
