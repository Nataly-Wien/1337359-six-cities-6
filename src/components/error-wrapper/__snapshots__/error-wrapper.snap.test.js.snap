// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should errors render correctly 1`] = `
<div>
  <div
    class="cities__status-wrapper"
  >
    <b
      class="cities__status"
    >
      Data loading error
    </b>
  </div>
</div>
`;
